### ISSUE TITLE
Utilize nonce from server response for subsequent client calls

### ISSUE THREAD

**Issue by Haza85** [Sphereon-Opensource/OID4VCI#110]
Hello, I've observed that the OpenID4VCIClient consistently utilizes the nonce from the access token acquisition response. However, there's a scenario where we need to make multiple sequential calls to the server to retrieve credentials using the same access token. In this case, each request requires a nonce from the preceding server response. Presently, the implementation relies on the same nonce for each request, resulting in the failure of subsequent requests due to nonce invalidity.

**Comment by nklomp** [comment]
Yeah you are correct, this is something we will be fixing after support for Draft 13 land, of which a first version will land this week
